cmd/kuberhealthy/kuberhealthy.go

```diff
@@ -629,6 +631,12 @@ func (k *Kuberhealthy) runCheck(ctx context.Context, c KuberhealthyCheck) {
 		}
 		log.Debugln("Done running check:", c.Name(), "in namespace", c.CheckNamespace())
 
+		// Record check run end time
+		// Subtract 10 seconds from run time since there are two 5 second sleeps during the check run where kuberhealthy
+		// waits for all pods to clear before running the check and waits for all pods to exit once the check has finished
+		// running. Both occur before and after the checker pod completes its run.
+		checkRunDuration := time.Now().Sub(checkStartTime) - time.Second * 10
```

Comment:
This worries me because if the upstream delays change in the future, then this logic will break and make bad times.  I think the right solution is to refactor the things that cause delay so that they don't do polling anymore.  That has not been an easy change in the past, though, so for now I think this will have to do...